Add/serverless mysql guide (#764)
zeit/docs: pages/guides/deploying-next-and-mysql-with-now.mdx

  height={380}
  oversize
/>

## Step 1: Populating Your MySQL Database

To use this guide you will need to have a remote MySQL database setup. There are many cloud providers who will be able to offer this service as a trial to you free of charge such as [Amazon Web Services](https://aws.amazon.com/rds/sqlserver/), [Google Cloud](https://cloud.google.com/sql/) and [Microsoft Azure](https://azure.microsoft.com/en-gb/services/sql-database/).

<Note>
  Please read the trial terms and conditions for each provider to avoid
  incurring unwanted charges.
</Note>

Once you have your remote MySQL database setup, you should make a note of your database credentials:

- Database name
- Database hostname
- Database username
- Database password

Using these credentials, you should connect to your database and insert the [example data](https://github.com/zeit/now-examples/blob/master/nextjs-mysql/db.sql) into a new table named `profiles`. This guide does not cover inserting records into a MySQL database, more information on doing this can be found in the [MySQL documentation](https://dev.mysql.com/doc/mysql-getting-started/en/#mysql-getting-started-connecting).

## Step 2: Set Up Your Project

> Now that you have populated the database, you should create a project directory and `cd` into it:

Reviewer: ```suggestion
Now that the database is populated, you can create a project directory and `cd` into it:
```
